driver/sql/persistence_strategy.go

```diff
@@ -1,11 +1,16 @@
 package sql
 
-import "github.com/hellofresh/goengine"
+import (
+	"github.com/hellofresh/goengine"
+	"github.com/hellofresh/goengine/metadata"
+)
 
 // PersistenceStrategy interface describes strategy of persisting messages in the database
 type PersistenceStrategy interface {
 	CreateSchema(tableName string) []string
+	EventColumnNames() []string
```

Comment:
Let's add a bit of documentation to this interface to make clear what this func is used for.
Also should change another name and rename `ColumnNames` to make clear that one is related to `PrepareSearch` and the other to `PrepareData`?

Reply:
https://github.com/hellofresh/goengine/pull/50/commits/61d74db0bbee23080d2fedca8c6828410c452479